src/client/entity-editor/submission-section/actions.ts

```diff
@@ -119,7 +119,11 @@ export function submit(
 	submissionUrl: string
 ): SubmitResult {
 	return (dispatch, getState) => {
-		const rootState = getState();
+		let rootState = getState();
+		dispatch(setSubmitted(true));
+		if (!rootState.getIn(['authorCreditEditor', 'n0', 'author'], null)) {
+			rootState = rootState.set('authorCreditEditor', Map({}));
+		}
```

Comment:
I'm not sure this is the way to go. Instead of an empty object representing an optional AC, I would prefer if there was a specific boolean passed along with the form data that can be used to ascertain if the AC should be optional (basically the same thing as what we do visually on the page).

That way for example if we one day have the ability to create entities via an API we can control this aspect more declaratively rather than rely on the ambiguous empty object vs. null